### PR TITLE
Remove Paragraph#paint

### PR DESCRIPTION
**Pull request by abarth**
We now use Canvas#drawParagraph.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: sky/engine/core/dart/painting.dart [hunk 1]
```diff
@@ -407,6 +407,9 @@ class Canvas extends NativeFieldWrapperClass2 {
   /// [PictureRecorder].
   void drawPicture(Picture picture) native "Canvas_drawPicture";
 
+  /// Draws the text in the given paragraph into this canvas at the given offset.
+  ///
+  /// Valid only after layout has been called on the paragraph.
```
Comment (Hixie) [inline sky/engine/core/dart/painting.dart]: `[Paragraph.layout]` ?

File: sky/engine/core/dart/text.dart [hunk 2]
```diff
@@ -612,11 +612,6 @@ abstract class Paragraph extends NativeFieldWrapperClass2 {
   /// function before painting or reading geometry from this paragraph.
   void layout() native "Paragraph_layout";
```
Comment (Hixie) [inline sky/engine/core/dart/text.dart]: The top of this class also mentions paint.

**Comment by Hixie** [comment]
lgtm

**Pull request merged**